src/main/atm2lndType.F90

```diff
@@ -80,6 +80,8 @@ module atm2lndType
      real(r8), pointer :: forc_vp_grc                   (:)   => null() ! atmospheric vapor pressure (Pa)
      real(r8), pointer :: forc_pco2_grc                 (:)   => null() ! CO2 partial pressure (Pa)
      real(r8), pointer :: forc_pco2_240_patch           (:)   => null() ! 10-day mean CO2 partial pressure (Pa)
+     real(r8), pointer :: forc_po3_grc                  (:)   => null() ! O3 partial pressure (Pa)
+     real(r8), pointer :: forc_po3_240_patch            (:)   => null() ! 10-day mean O3 partial pressure (Pa)
```

Comment:
You are right. It is not used. I originally intended to use it as a diagnostic, because fumigation experiments often report stomata conductance and photosynthesis in such a manner. I will remove it for now.